cm_bpyNodes.py

```diff
@@ -478,7 +478,7 @@ class FilterNode(LogicNode):
 
     def draw_buttons(self, context, layout):
         layout.prop(self, "Operation")
-        if (self.Operation == "EQUAL" or self.Operation == "NOT EQUAL" or self.Operation == "LESS" or self.Operation == "GREATER"):
+        if self.Operation in ["EQUAL", "NOT EQUAL", "LESS", "GREATER"]:
```

Comment:
Good question, I will ask a [question](http://stackoverflow.com/questions/39960941/python-best-practice-series-of-ors-or-in) on SO about it. In the mean time I found this in the PEP-8 guidelines:

> ...code is read much more often than it is written. The guidelines provided here are intended to improve the readability of code...

> However, know when to be inconsistent -- sometimes style guide recommendations just aren't applicable. When in doubt, use your best judgment. Look at other examples and decide what looks best.